app/javascript/utilities/showModal.jsx

```diff
@@ -49,14 +46,14 @@ export const showWindowModal = async ({
 
   render(
     <Modal
-      title={title}
       onClose={() => {
         render(null, currentModalContainer);
       }}
-      size={size}
       focusTrapSelector={`#${WINDOW_MODAL_ID}`}
+      {...modalProps}
     >
       <div
+        className="h-100 w-100"
```

Comment:
Weirdly, this extra `div` we use was causing issues in that the internals of it weren't able to expand any bigger than it's auto sizing 🙃 

I had a quick check around, and I don't believe this has any negative impact, it just means the container we pop our modal content into matches the size of the `Modal` component content area